content/references/markdown/index.md

```diff
@@ -4,12 +4,171 @@ topic: markdown
 next: /references/markdown/extensions/
 ---
 
+
+
 # Basic Markdown Syntax
 
 Codewars supports [Markdown][wiki-markdown].
 
 More specifically, [GitHub Flavored Markdown][gfm] (strict superset of [CommonMark][common-mark]) and few [Codewars extensions][extensions].
 
+You may find below a summary of the most used ones.
+
+
+
+
+## Emphasis
+
+```markdown
+_This is italic_
+*This is also italic*
+
+__This is bold__
+**This is bold too**
+
+___here, italic and bold___
+***here, italic and bold***
+```
+Note that `*You **can** combine them*`: *You **can** combine them*.
+
+
+
+
+## Headers
+
+<pre><code>
+# This is &lt;h1&gt;
+## This is &lt;h2&gt;
+### This is &lt;h3&gt;
+</code></pre>
+
+
+
+## Displaying code
+
+Use code spans and code blocks to show the text as is. Use code block to show the syntax highlighted code.
+
+### Code span
+
+Use single backtick `` ` `` to show the text as is inline.
```

Comment:
```suggestion
Use single backtick (\`) to show the text as is inline.
```

Reply:
This is because the current code span is style with surrounding backticks. Using code span will show three backticks.